IFC4x3/PropertyConstants/t/TURNOUTPANEL_1G$HCil_D2dOSmwDaZucA0.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="TURNOUTPANEL_1GHCil_D2dOSmwDaZucA0" Name="TURNOUTPANEL" UniqueId="50fd132c-bfe3-429d-8730-e8d923e26280" />
+
```

Comment:
No documentation, even if it is maybe not entirely necessary here.

Reply:
erased. Comming from Element assembly. Commit 6cb18533994026ae118add924a2b65d2f98b8472